### ISSUE TITLE
XmlSchema information from package-info.java is lost in swagger.json

### ISSUE THREAD

**Issue by jskovjyskebankdk** [stoicflame/enunciate#727]
We have a namespace for DTOs, expressed in package-info.java:

````
@javax.xml.bind.annotation.XmlSchema(namespace="http://www.jyskebank.dk/domain-meta/version/1")
package foo.bar.dto;
````

This information is not expressed in swagger.json, preventing generation of java DTO code from swagger.json (because they lack the namespace information).

(not sure if it is possible to express in swagger.json)

**Comment by stoicflame** [comment]
Based on what I see [documented here](https://swagger.io/specification/#xmlObject), it's unclear how the namespace gets specified at the "root" level. Applying the "xml" property at the "root" level _might_ work, but needs to be tested:

```json
 "root_element" : {
      "type" : "object",
      "xml" : {
        "namespace" : "http://www.jyskebank.dk/domain-meta/version/1"
      },
    }
```

**Comment by jskovjyskebankdk** [comment]
I also failed to find anything in the spec.

But now I found https://github.com/swagger-api/swagger-core/issues/2150, so it should be possible somehow. I guess I'll try to deploy swagger with my app, to see what it generates at run-time. That should settle it.

**Comment by jskovjyskebankdk** [comment]
FWIW your suggestion just adds a new DTO class.

**Comment by jskovjyskebankdk** [comment]
Uh, but that may just be the java client generator code.
No effect when adding namespace anywhere. So your suggestion may actually work.
(I now get that you want to add namespace to the root of each type, not add a magically named root_element :)

**Comment by stoicflame** [comment]
Included in the fix for #726.
